nodejs/README.md

```diff
@@ -2,55 +2,54 @@
 
 ## Installing
 
-The tutorials are using CNCF Jaeger (https://github.com/jaegertracing/jaeger) as the tracing backend, 
+The tutorials are using CNCF Jaeger (https://github.com/jaegertracing/jaeger) as the tracing backend,
 [see here](../README.md) how to install it in a Docker image.
 
-Install the dependencies:
+First clone the [opentracing-tutorial](https://github.com/yurishkuro/opentracing-tutorial), then install the dependencies:
 
 ```
 cd opentracing-tutorial/nodejs
 npm install
 ```
 
-The rest of the commands in the Node.js tutorials should be executed relative to this directory.
+The rest of the commands in the Node.js tutorials should be executed relative to this `nodejs` directory.
 
 ## Under construction
 
 This tutorial is currently incomplete. You can try following the tutorials in the other languages
 and adapt them to Node.js, as all the same features are available in the OpenTracing API for Javascript
-(https://github.com/opentracing/opentracing-javascript).  Use the following code to initialize Jaeger tracer:
+(https://github.com/opentracing/opentracing-javascript). Use the following code to initialize Jaeger tracer:
 
 ```javascript
-var initJaegerTracer = require('jaeger-client').initTracer;
+var initJaegerTracer = require("jaeger-client").initTracer;
```

Comment:
what's the reason for changing to double-quotes?

Reply:
Formatting; should make it consistent with all the other require statements (at least in Lessons 1 and 2).